src/Summoner/Template.hs

```diff
@@ -369,7 +370,8 @@ createStackTemplate ProjectData{..} = Dir (toString repo) $
     -- create travis.yml template
     travisYml :: Text
     travisYml =
-        let travisMtr = T.concat (map (travisMatrixItem . showGhcVer) (delete defaultGHC testedVersions))
+        let travisStackMtr = T.concat (map (travisStackMatrixItem . showGhcVer) (delete defaultGHC testedVersions))
```

Comment:
I think you can make this line more consistent with `cabal` matrix item and move `showGhcVer` as a `ViewPattern` inside `travisStackMatrixItem` function.